website/docs/concepts/router.mdx

```diff
@@ -94,7 +95,7 @@ fn switch(routes: Route) -> Html {
 fn app() -> Html {
     html! {
         <BrowserRouter>
-            <Switch<Route> render={switch} />
+            <Switch<Route> render={switch} /> // <- depends on <BrowserRouter>
```

Comment:
```suggestion
            <Switch<Route> render={switch} /> // <- must be child of <BrowserRouter>
```

Reply:
This modified wording, imo, might lead newcomers to add a router to every component they use links in. This is obviously not the intended usage. Should it mention perhaps:

> You usually only need a single `<BrowserRouter />` in your app, most often rendered immediately by your most top-level `<App />` component.

?

Reply:
Agreed. There's no dependency so I don't want to use "depends on". A comment in the snippet or a `:::note` stating there should be one router should be good